miner/src/miner.rs

```diff
@@ -23,9 +19,9 @@ pub struct Miner {
     pub works: LruCache<H256, Work>,
     pub worker_controllers: Vec<WorkerController>,
     pub work_rx: Receiver<Work>,
-    pub seal_rx: Receiver<(H256, Seal)>,
+    pub nonce_rx: Receiver<(H256, u64)>,
     pub pb: ProgressBar,
-    pub seals_found: u64,
+    pub nonces_found: u64,
```

Comment:
Is `nonces_found` the same as blocks found? I'm not sure which we should display in the progress, blocks found or nonces tried.

Reply:
it's same as blocks found